src/initializers/kafka/kafka.ts

```diff
@@ -0,0 +1,73 @@
+import { getLogger } from '../log4js';
+import { NProducer, NConsumer } from 'sinek';
+import authOptions from './auth-options';
+
+export default class Kafka {
+  private producer: NProducer;
+  private options: any;
+  private authOptions: any;
+
+  constructor(options: {
+    key: string;
+    cert: string;
+    ca: string;
+    groupId: string;
+    clientId: string;
+    brokers: string[];
+  }) {
+    this.options = options;
+  }
+
+  public async connect() {
+    const { key, cert, ca } = this.options;
+    this.authOptions = authOptions({ key, cert, ca });
+    this.producer = this.createProducer();
+    this.producer.on('error', err => getLogger('orka.kafka.connect').error(err));
+    await this.producer.connect();
+  }
+
+  public async send(topic: string, message: string | Buffer) {
+    const { key, partition, offset } = await this.producer.send(topic, message);
+    getLogger('orka.kafka.send').info(`partition(${partition})[${offset}][${key}] produced for topic ${topic}`);
+  }
+
+  public createConsumer(topic: string) {
+    const { groupId, brokers } = this.options;
+    const config = {
+      groupId,
+      logger: getLogger('orka.kafka.consumer.internal'),
+      noptions: {
+        'group.id': groupId,
+        'metadata.broker.list': brokers.join(','),
+        'socket.keepalive.enable': true,
```

Comment:
`Socket keepalives need to be enabled on both sides to expedite connection termination on connectivity loss (e.g., network partitioning) for both parties. This means the broker needs to implement it too.` Does it?

Reply:
I can't find anything related in lenses. I found in the issues section that it doesn't cause any harm if it's enabled. We can check further if you want, but I'll need assistance regarding the lenses/kafka configuration.

Reply:
Thanks for checking.